examples/cis2-multi/src/lib.rs

```diff
@@ -43,31 +188,17 @@ pub type ContractTokenId = TokenIdU8;
 /// Contract token amount type.
 pub type ContractTokenAmount = TokenAmountU64;
 
-#[derive(Serialize, SchemaType)]
-pub struct MintParam {
-    pub token_amount: ContractTokenAmount,
-    pub metadata_url: MetadataUrl,
-}
-
-/// The parameter for the contract function `mint` which mints a number of
-/// token types and/or amounts of tokens to a given address.
+/// The parameter for the contract function `mint` which mints/airdrops a number
+/// of tokens to the owner's address.
 #[derive(Serialize, SchemaType)]
 pub struct MintParams {
     /// Owner of the newly minted tokens.
-    pub owner:  Address,
-    /// A collection of tokens to mint.
-    pub tokens: collections::BTreeMap<ContractTokenId, MintParam>,
-}
-
-/// The parameter type for the contract function `setImplementors`.
-/// Takes a standard identifier and a list of contract addresses providing
-/// implementations of this standard.
-#[derive(Debug, Serialize, SchemaType)]
-pub struct SetImplementorsParams {
-    /// The identifier for the standard.
-    id:           StandardIdentifierOwned,
-    /// The addresses of the implementors of the standard.
-    implementors: Vec<ContractAddress>,
+    pub owner:        Address,
+    /// The metadata_url of the token (is ignored except for the first time
+    /// a token_id is minted).
```

Comment:
I assume it is resolved given your below comment.

Reply:
I would prefer if this section did not mention the "ignored" since it's not really relevant here (which makes it hard to understand). But it's a minor point.